packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -836,14 +839,26 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         private _storage?: IDocumentStorageService,
     ) {
         super();
-
+        this.metadata = metadata ?? this.formMetadata();
         this.baseSummaryMessage = metadata?.message;
         /**
           * gcFeature in metadata is introduced with v1 in the metadata blob. Forced to 0/disallowed before that.
           * For existing documents, we get this value from the metadata blob.
           * For new documents, we get this value based on the gcAllowed flag in runtimeOptions.
           */
         const prevSummaryGCVersion = existing ? getGCVersion(metadata) : undefined;
+
+        /**
+         * If container exists, then get these values from metadata.
+         * If there is no metadata or metadata does not have these values, it means this is an old document,
+         * so these values will be undefined. If container doesn't exist, we initialize these values
+         */
+        this.metadata.createContainerRuntimeVersion = existing ? metadata?.createContainerRuntimeVersion ?? undefined
```

Comment:
What "metadata?.createContainerRuntimeVersion ?? undefined" does? Is it to convert "" to undefined?
If so, why, and when can we have empty string?

Reply:
Actually, I believe ?? will not touch "", so this seems to be noop?